### ISSUE TITLE
Support Quantized Tensorflow lite model

### ISSUE THREAD

**Issue by kris-himax** [ultralytics/hub#97]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and [discussions](https://github.com/ultralytics/hub/discussions) and found no similar questions.

### Question

Hi,
The environment is amazing.
But the TensorFlow Lite model is only supported floating.
We will deploy the tflite model to the devices which are resource constrained.
Could you support quantized TensorFlow Lite model with uint8 or int8?

Thanks

**Comment by kris-himax** [comment]
@glenn-jocher thanks for the reply!
I am the member at the Himax Technologies, Inc (https://www.himax.com.tw/) which are researching and developing next generation ultra low power deep learning IC with M55 and U55.
We are really interested in your model with no code environment to support quantized TFlite int models.

**Comment by glenn-jocher** [comment]
@kris-himax got it! One solution you could use is to download the PyTorch model instead and export it to TFLite INT8 using YOLOv5, i.e.

```bash
git clone https://github.com/ultralytics/yolov5  # clone
cd yolov5
pip install -r requirements.txt  # install

python export.py --weights model.pt --include tflite  # FP16
python export.py --weights model.pt --include tflite --int8  # INT8
```

One problem we've seen though is that INT8 models do not produce identical validation results. FP32 and FP16 models will produce identical mAP, but quantizing to INT8 results in some drop, so you should definitely validate both exports first to verify you are ok with the accuracy drop.

Also if you have any ideas or solutions to fixing the INT8 validation drop please let us know, thank you!

**Comment by kris-himax** [comment]
@glenn-jocher hi,
While we try to quantize the other model which the activation function uses silu, the result of quantized int8 model is worse.
In our opinion, maybe the reason of  quantized  INT8 results dropping is the silu activation function.
Would you try to replace silu to other activation function and then do the quantization?

Thanks

**Comment by glenn-jocher** [comment]
@kris-himax TFLite seems to suffer from poor results at INT8, likely due to their specific quantization strategy. Other formats like CoreML suffer no significant drop in mAP when moving from FP16 to INT8.

BTW I just added the ability to add any activation to YOLOv5 models with a new `activation:` field in a model yaml. This works with any PyTorch activation, including ones that require arguments. See https://github.com/ultralytics/yolov5/pull/9371 for more details.
